sdk/schemaregistry/schema-registry-avro/test/schemaRegistryAvroSerializer.spec.ts

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SchemaRegistryAvroSerializer } from "../src";
+import { assert, use as chaiUse } from "chai";
+import * as avro from "avsc/";
+import chaiPromises from "chai-as-promised";
+
+import {
+  GetSchemaByIdOptions,
+  GetSchemaIdOptions,
+  RegisterSchemaOptions,
+  Schema,
+  SchemaDescription,
+  SchemaId,
+  SchemaRegistry
+} from "@azure/schema-registry";
+
+chaiUse(chaiPromises);
+
+const testSchemaObject = {
+  type: "record",
+  name: "AvroUser",
+  namespace: "com.azure.schemaregistry.samples",
+  fields: [
+    {
+      name: "name",
+      type: "string"
+    },
+    {
+      name: "favoriteNumber",
+      type: "int"
+    }
+  ]
+};
+
+const testSchemaIds = [
+  "{773E17BE-793E-40B0-98F1-0A6EA3C11895}",
+  "{DC7EF290-CDB1-4245-8EE8-3DD52965866E}"
+].map((x) => x.replace(/[\{\-\}]/g, ""));
+
+const testSchema = JSON.stringify(testSchemaObject);
+const testValue = { name: "Nick", favoriteNumber: 42 };
+const testAvroType = avro.Type.forSchema(<any>testSchemaObject);
+
+describe("SchemaRegistryAvroSerializer", function() {
+  it("captures constructor args", () => {
```

Comment:
this test feels maybe a little silly, but I suppose it gives coverage

Reply:
It's definitely silly. I got trolled by the coverage numbers in my face on every npm run test. 🤣

I'm just going to delete this test and the public properties.

Reply:
I also may have written some silly tests in order to win the code coverage game before... 😏

There is no judgement here, only amusement. 😉